staging/src/k8s.io/cluster-bootstrap/token/api/types.go

```diff
@@ -96,7 +96,7 @@ const (
 	BootstrapGroupPattern = `\Asystem:bootstrappers:[a-z0-9:-]{0,255}[a-z0-9]\z`
 
 	// BootstrapTokenPattern defines the {id}.{secret} regular expression pattern
-	BootstrapTokenPattern = `\A([a-z0-9]{6})\.([a-z0-9]{16})\z`
+	BootstrapTokenPattern = `\A([a-z0-9]{6})\.([a-z0-9]{24})\z`
```

Comment:
this is an incompatible change that will break all callers of ParseToken / IsValidBootstrapToken / etc

Reply:
if so, do we still want to fix the low entropy bootstrap tokens problem?